doc/connectivity/bluetooth/api/mesh/access.rst

```diff
@@ -171,9 +171,9 @@ Composition Data
 .. note::
 
    The implementation of the Bluetooth Mesh Protocol Specification version 1.1
-   is currently in an experimental state. For Bluetooth Mesh Profile Specification
+   is currently in an experimental state. For Bluetooth Mesh Protocol Specification
```

Comment:
1.0.1 is still called `Profile`: https://www.bluetooth.com/specifications/specs/mesh-profile-1-0-1/ I guess we can't change this